jerry-core/parser/js/js-parser-util.c

```diff
@@ -923,7 +923,7 @@ parser_error_to_string (parser_error_t error) /**< error code */
     }
     case PARSER_ERR_NON_STRICT_ARG_DEFINITION:
     {
-      return "Non strict argument definition.";
+      return "Non-strict argument definition.";
```

Comment:
Strict is enabled by "use strict" and not "use-strict" so perhaps we shouldn't add dashes.